nf_core/pipeline-template/nextflow.config

```diff
@@ -186,18 +186,11 @@ profiles {
 
 {% if nf_core_configs -%}
 // Load nf-core custom profiles from different Institutions
-try {
-    includeConfig "${params.custom_config_base}/nfcore_custom.config"
-} catch (Exception e) {
-    System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
-}
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
 // Load {{ name }} custom profiles from different institutions.
-try {
-    includeConfig "${params.custom_config_base}/pipeline/{{ short_name }}.config"
-} catch (Exception e) {
-    System.err.println("WARNING: Could not load nf-core/config/{{ short_name }} profiles: ${params.custom_config_base}/pipeline/{{ short_name }}.config")
-}
+// TODO nf-core: Optionally, you can add a pipeline-specific nf-core config at https://github.com/nf-core/configs
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/{{ short_name }}.config" : "/dev/null"
```

Comment:
> I would consider commenting out these lines on the template again and let users decide if they want to include a pipeline specific config.

Agree 👍🏻 

```suggestion
// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/{{ short_name }}.config" : "/dev/null"
```

Reply:
Does this mean that a pipeline developer has to do this? 

If so I don't like that idea so much as it never gets uncommented (as much people don't know about it), but then you have users come along asking for the support which then requires a whole new release of the pipeline. 

We specifically added automatically generated the 'dummy' configs to nf-core/configs recently so it's a seamless experience. A pipeline developer just needs to wait overnight for the dummy configs to be added (with the caveat the pipeline has to be under the nf-core organisation). 

The other solution I can think to get around this: is to comment it out, but make a release linting _error_ that it must be uncommented before release. Thus from first release it's supported

Reply:
The problem now is that without the try/except it will fail right after you use `nf-core pipelines create`, because the config is not added yet to nf-core. 
The release linting error sounds like a good solution to me :)

Reply:
linting test added 🙂